log4j missing dependency error

ADE gives the following error:
```
[INFO] Reactor Summary for ade 1.0.4:
[INFO] 
[INFO] ade ................................................ SUCCESS [  3.089 s]
[INFO] ade-core ........................................... SUCCESS [01:12 min]
[INFO] ade-ext ............................................ SUCCESS [ 33.735 s]
[INFO] ade-assembly ....................................... FAILURE [  0.520 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:49 min
[INFO] Finished at: 2022-06-22T10:37:34-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ade-assembly: Could not resolve dependencies for project org.openmainframe.ade:ade-assembly:pom:1.0.4: Could not find artifact log4j:log4j:jar:2.15.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

To reproduce:
```
mvn clean package
```

--- a/ade-assembly/pom.xml
+++ b/ade-assembly/pom.xml
@@ -28,11 +28,15 @@
       <version>1.7.12</version>
     </dependency>
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-1.2-api</artifactId>
       <version>2.15.0</version>
-      <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <version>2.15.0</version>
+    </dependency>    
   </dependencies>
   <build>
     <plugins>

